{"updated_at": "2012-12-28T19:47:30Z", "repo": "rails", "created_at": "2012-12-28T19:47:13Z", "owner": "rails", "url": "https://api.github.com/repos/rails/rails/issues/comments/11739888", "id": 11739888, "issue_url": "https://api.github.com/repos/rails/rails/issues/8643", "issue_id": 8643, "user": {"repos_url": "https://api.github.com/users/steveklabnik/repos", "type": "User", "html_url": "https://github.com/steveklabnik", "starred_url": "https://api.github.com/users/steveklabnik/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/steveklabnik/subscriptions", "following_url": "https://api.github.com/users/steveklabnik/following{/other_user}", "gravatar_id": "233c279c012ebac792aaa805f966cbc7", "organizations_url": "https://api.github.com/users/steveklabnik/orgs", "url": "https://api.github.com/users/steveklabnik", "gists_url": "https://api.github.com/users/steveklabnik/gists{/gist_id}", "avatar_url": "https://0.gravatar.com/avatar/233c279c012ebac792aaa805f966cbc7?d=https%3A%2F%2Fidenticons.github.com%2F0c22d45f31acf0f4192c8c12b7a55dee.png", "id": 27786, "events_url": "https://api.github.com/users/steveklabnik/events{/privacy}", "login": "steveklabnik", "received_events_url": "https://api.github.com/users/steveklabnik/received_events", "followers_url": "https://api.github.com/users/steveklabnik/followers"}, "body": "Two things: How is adding this file removing the warning? Wouldn't this add a definition? It's not that I don't believe you, it's that I don't see it.\r\n\r\nSecondly, I'm not entirely sure this belongs inside the begin block: we know it will be always loaded, since it's also ActiveSupport.", "_id": {"$oid": "523d79a8bd354364b90071a9"}, "html_url": "https://github.com/rails/rails/pull/8643#issuecomment-11739888"}
